api/custom_auth/jwt_cookie/signals.py

```diff
@@ -0,0 +1,20 @@
+from typing import Any
+from urllib.parse import urlparse
+
+from core.helpers import get_current_site_url
+from corsheaders.signals import check_request_enabled
+from django.dispatch import receiver
+from django.http import HttpRequest
+
+
+@receiver(check_request_enabled)
+def cors_allow_current_site(request: HttpRequest, **kwargs: Any) -> bool:
+    # The signal is expected to only be dispatched:
```

Comment:
Is there some documentation that we can link to for this?

Reply:
Not really, you have to read django-cors-headers source for this knowledge.

Reply:
Understood.